### PR TITLE
HBASE-23187 Update parent region state to SPLIT in meta

### PR DESCRIPTION
**Pull request by binlijin**
When split, the parent region set to SPLIT in memory, but the meta do not, so in some circumstances will bring back the parent region.

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| :blue_heart: |  reexec  |   3m 29s |  Docker mode activated.  |
||| _ Prechecks _ |
| :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ master Compile Tests _ |
| :blue_heart: |  mvndep  |   0m 33s |  Maven dependency ordering for branch  |
| :green_heart: |  mvninstall  |   5m 50s |  master passed  |
| :green_heart: |  compile  |   1m 24s |  master passed  |
| :green_heart: |  checkstyle  |   2m  1s |  master passed  |
| :green_heart: |  shadedjars  |   4m 58s |  branch has no errors when building our shaded downstream artifacts.  |
| :green_heart: |  javadoc  |   0m 57s |  master passed  |
| :blue_heart: |  spotbugs  |   4m 27s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| :green_heart: |  findbugs  |   5m 35s |  master passed  |
||| _ Patch Compile Tests _ |
| :blue_heart: |  mvndep  |   0m 14s |  Maven dependency ordering for patch  |
| :green_heart: |  mvninstall  |   5m 28s |  the patch passed  |
| :green_heart: |  compile  |   1m 23s |  the patch passed  |
| :green_heart: |  javac  |   1m 23s |  the patch passed  |
| :broken_heart: |  checkstyle  |   1m 28s |  hbase-server: The patch generated 20 new + 0 unchanged - 0 fixed = 20 total (was 0)  |
| :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| :green_heart: |  shadedjars  |   4m 59s |  patch has no errors when building our shaded downstream artifacts.  |
| :green_heart: |  hadoopcheck  |  17m 25s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2.  |
| :green_heart: |  javadoc  |   0m 57s |  the patch passed  |
| :green_heart: |  findbugs  |   6m 12s |  the patch passed  |
||| _ Other Tests _ |
| :green_heart: |  unit  |   1m 54s |  hbase-client in the patch passed.  |
| :broken_heart: |  unit  | 247m 14s |  hbase-server in the patch failed.  |
| :green_heart: |  asflicense  |   0m 50s |  The patch does not generate ASF License warnings.  |
|  |   | 320m 22s |   |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.security.access.TestSnapshotScannerHDFSAclController |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.3 Server=19.03.3 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/732 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux a659bf23e03e 4.15.0-65-generic #74-Ubuntu SMP Tue Sep 17 17:06:04 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-732/out/precommit/personality/provided.sh |
| git revision | master / 946f1e9e25 |
| Default Java | 1.8.0_181 |
| checkstyle | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/1/artifact/out/diff-checkstyle-hbase-server.txt |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/1/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/1/testReport/ |
| Max. process+thread count | 4975 (vs. ulimit of 10000) |
| modules | C: hbase-client hbase-server U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/1/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.11.0 https://yetus.apache.org |


This message was automatically generated.

File: hbase-client/src/main/java/org/apache/hadoop/hbase/MetaTableAccessor.java [hunk 1]
```diff
@@ -1580,6 +1580,7 @@ public static void splitRegion(Connection connection, RegionInfo parent, long pa
       Put putParent = makePutFromRegionInfo(RegionInfoBuilder.newBuilder(parent)
                         .setOffline(true)
                         .setSplit(true).build(), time);
+      addRegionStateToPut(putParent, State.SPLIT);
```
Comment (Apache9) [inline hbase-client/src/main/java/org/apache/hadoop/hbase/MetaTableAccessor.java]: I think for split, it is SPLIT_PARENT? And what is the way we do this for branch-1? In the above code, the RegionInfo has been set to offline and also split, so I do not think it should come back online again? If it does then there are some bugs in other places.

Comment (binlijin) [reply hbase-client/src/main/java/org/apache/hadoop/hbase/MetaTableAccessor.java]: The SPLIT_PARENT add back when enable table, and the code is in: 
https://github.com/apache/hbase/blob/master/hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/EnableTableProcedure.java#L114
https://github.com/apache/hbase/blob/master/hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/RegionStates.java#L329
https://github.com/apache/hbase/blob/master/hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/RegionStates.java#L349
And i not am familiar with all this logic.

**Comment by Apache9** [comment]
I'm also not very familiar with the details SPLIT/MERGE processings. Let me take a look on it, we need to find the correct way to fix this.

**Comment by Apache9** [comment]
Anyway, I think we could persist the SPLIT state but the problem is that, when closing the parent region we will set the state to CLOSED. So for fixing this issue, I think we could add a check here

https://github.com/apache/hbase/blob/master/hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/RegionStates.java#L349

which returns false if the RegionInfo.isSplit is true.

And can file another issue to just remove the usage of SPLIT, SPLITTING and SPLITTING_NEW, as the state of SplitTableProcedure is enough?

**Comment by binlijin** [comment]
* Utility. Whether to include region in list of regions. Default is to
   * weed out split and offline regions.
   * @return True if we should include the <code>node</code> (do not include
   * if split or offline unless <code>offline</code> is set to true.

As in the comment, when offline==true will include if split or offline, do you know when this is needed?  
@Apache9

**Comment by Apache9** [comment]
The problem here is that we need to check the split and offline field in RegionInfo, not the state...

**Comment by Apache9** [comment]
Or we could check for OFFLINE state but for at least for split, we should check the field in RegionInfo.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| :blue_heart: |  reexec  |   1m  6s |  Docker mode activated.  |
||| _ Prechecks _ |
| :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ master Compile Tests _ |
| :blue_heart: |  mvndep  |   0m 34s |  Maven dependency ordering for branch  |
| :green_heart: |  mvninstall  |   6m  4s |  master passed  |
| :green_heart: |  compile  |   1m 27s |  master passed  |
| :green_heart: |  checkstyle  |   2m  2s |  master passed  |
| :green_heart: |  shadedjars  |   5m  0s |  branch has no errors when building our shaded downstream artifacts.  |
| :green_heart: |  javadoc  |   0m 57s |  master passed  |
| :blue_heart: |  spotbugs  |   4m 16s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| :green_heart: |  findbugs  |   5m 27s |  master passed  |
||| _ Patch Compile Tests _ |
| :blue_heart: |  mvndep  |   0m 14s |  Maven dependency ordering for patch  |
| :green_heart: |  mvninstall  |   5m 26s |  the patch passed  |
| :green_heart: |  compile  |   1m 22s |  the patch passed  |
| :green_heart: |  javac  |   1m 22s |  the patch passed  |
| :broken_heart: |  checkstyle  |   1m 28s |  hbase-server: The patch generated 21 new + 8 unchanged - 0 fixed = 29 total (was 8)  |
| :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| :green_heart: |  shadedjars  |   5m  1s |  patch has no errors when building our shaded downstream artifacts.  |
| :green_heart: |  hadoopcheck  |  17m 14s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2.  |
| :green_heart: |  javadoc  |   0m 57s |  the patch passed  |
| :green_heart: |  findbugs  |   5m 41s |  the patch passed  |
||| _ Other Tests _ |
| :green_heart: |  unit  |   1m 49s |  hbase-client in the patch passed.  |
| :broken_heart: |  unit  | 257m 12s |  hbase-server in the patch failed.  |
| :green_heart: |  asflicense  |   0m 46s |  The patch does not generate ASF License warnings.  |
|  |   | 327m 22s |   |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.client.TestFromClientSideWithCoprocessor |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.4 Server=19.03.4 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/732 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux cedaf3d92a24 4.15.0-65-generic #74-Ubuntu SMP Tue Sep 17 17:06:04 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-732/out/precommit/personality/provided.sh |
| git revision | master / da9a53ed4d |
| Default Java | 1.8.0_181 |
| checkstyle | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/2/artifact/out/diff-checkstyle-hbase-server.txt |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/2/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/2/testReport/ |
| Max. process+thread count | 4722 (vs. ulimit of 10000) |
| modules | C: hbase-client hbase-server U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/2/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.11.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| :blue_heart: |  reexec  |   1m 48s |  Docker mode activated.  |
||| _ Prechecks _ |
| :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ master Compile Tests _ |
| :blue_heart: |  mvndep  |   1m  7s |  Maven dependency ordering for branch  |
| :green_heart: |  mvninstall  |   7m  0s |  master passed  |
| :green_heart: |  compile  |   1m 25s |  master passed  |
| :green_heart: |  checkstyle  |   1m 51s |  master passed  |
| :green_heart: |  shadedjars  |   4m 37s |  branch has no errors when building our shaded downstream artifacts.  |
| :green_heart: |  javadoc  |   1m  1s |  master passed  |
| :blue_heart: |  spotbugs  |   4m 11s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| :green_heart: |  findbugs  |   5m 16s |  master passed  |
||| _ Patch Compile Tests _ |
| :blue_heart: |  mvndep  |   0m 16s |  Maven dependency ordering for patch  |
| :green_heart: |  mvninstall  |   4m 50s |  the patch passed  |
| :green_heart: |  compile  |   1m 23s |  the patch passed  |
| :green_heart: |  javac  |   1m 23s |  the patch passed  |
| :broken_heart: |  checkstyle  |   1m 18s |  hbase-server: The patch generated 20 new + 6 unchanged - 2 fixed = 26 total (was 8)  |
| :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| :green_heart: |  shadedjars  |   4m 36s |  patch has no errors when building our shaded downstream artifacts.  |
| :green_heart: |  hadoopcheck  |  15m 41s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2.  |
| :green_heart: |  javadoc  |   0m 58s |  the patch passed  |
| :green_heart: |  findbugs  |   5m 17s |  the patch passed  |
||| _ Other Tests _ |
| :green_heart: |  unit  |   1m 50s |  hbase-client in the patch passed.  |
| :broken_heart: |  unit  | 262m 54s |  hbase-server in the patch failed.  |
| :green_heart: |  asflicense  |   1m  6s |  The patch does not generate ASF License warnings.  |
|  |   | 331m 21s |   |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.client.TestFromClientSide |
|   | hadoop.hbase.client.TestFromClientSideWithCoprocessor |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.4 Server=19.03.4 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/732 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 6f75012a2f65 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-732/out/precommit/personality/provided.sh |
| git revision | master / 42d535a57a |
| Default Java | 1.8.0_181 |
| checkstyle | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/3/artifact/out/diff-checkstyle-hbase-server.txt |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/3/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/3/testReport/ |
| Max. process+thread count | 4926 (vs. ulimit of 10000) |
| modules | C: hbase-client hbase-server U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/3/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.11.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| :blue_heart: |  reexec  |   3m 34s |  Docker mode activated.  |
||| _ Prechecks _ |
| :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ master Compile Tests _ |
| :blue_heart: |  mvndep  |   0m 36s |  Maven dependency ordering for branch  |
| :green_heart: |  mvninstall  |   5m 17s |  master passed  |
| :green_heart: |  compile  |   1m 21s |  master passed  |
| :green_heart: |  checkstyle  |   1m 50s |  master passed  |
| :green_heart: |  shadedjars  |   4m 36s |  branch has no errors when building our shaded downstream artifacts.  |
| :green_heart: |  javadoc  |   0m 58s |  master passed  |
| :blue_heart: |  spotbugs  |   4m  4s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| :green_heart: |  findbugs  |   5m 10s |  master passed  |
||| _ Patch Compile Tests _ |
| :blue_heart: |  mvndep  |   0m 15s |  Maven dependency ordering for patch  |
| :green_heart: |  mvninstall  |   4m 55s |  the patch passed  |
| :green_heart: |  compile  |   1m 21s |  the patch passed  |
| :green_heart: |  javac  |   1m 21s |  the patch passed  |
| :green_heart: |  checkstyle  |   0m 33s |  The patch passed checkstyle in hbase-client  |
| :green_heart: |  checkstyle  |   1m 17s |  hbase-server: The patch generated 0 new + 6 unchanged - 2 fixed = 6 total (was 8)  |
| :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| :green_heart: |  shadedjars  |   4m 39s |  patch has no errors when building our shaded downstream artifacts.  |
| :green_heart: |  hadoopcheck  |  15m 40s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2.  |
| :green_heart: |  javadoc  |   0m 59s |  the patch passed  |
| :green_heart: |  findbugs  |   5m 25s |  the patch passed  |
||| _ Other Tests _ |
| :green_heart: |  unit  |   1m 51s |  hbase-client in the patch passed.  |
| :broken_heart: |  unit  | 280m  8s |  hbase-server in the patch failed.  |
| :green_heart: |  asflicense  |   1m  3s |  The patch does not generate ASF License warnings.  |
|  |   | 348m  9s |   |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.client.TestSnapshotTemporaryDirectoryWithRegionReplicas |
|   | hadoop.hbase.client.TestFromClientSideWithCoprocessor |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.4 Server=19.03.4 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/4/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/732 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 3833051f0f2c 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-732/out/precommit/personality/provided.sh |
| git revision | master / 4c75485f11 |
| Default Java | 1.8.0_181 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/4/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/4/testReport/ |
| Max. process+thread count | 5008 (vs. ulimit of 10000) |
| modules | C: hbase-client hbase-server U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/4/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.11.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| :blue_heart: |  reexec  |   1m 12s |  Docker mode activated.  |
||| _ Prechecks _ |
| :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ master Compile Tests _ |
| :blue_heart: |  mvndep  |   0m 33s |  Maven dependency ordering for branch  |
| :green_heart: |  mvninstall  |   5m 45s |  master passed  |
| :green_heart: |  compile  |   1m 21s |  master passed  |
| :green_heart: |  checkstyle  |   2m  3s |  master passed  |
| :green_heart: |  shadedjars  |   4m 59s |  branch has no errors when building our shaded downstream artifacts.  |
| :green_heart: |  javadoc  |   0m 57s |  master passed  |
| :blue_heart: |  spotbugs  |   4m 38s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| :green_heart: |  findbugs  |   5m 44s |  master passed  |
||| _ Patch Compile Tests _ |
| :blue_heart: |  mvndep  |   0m 14s |  Maven dependency ordering for patch  |
| :green_heart: |  mvninstall  |   5m 30s |  the patch passed  |
| :green_heart: |  compile  |   1m 21s |  the patch passed  |
| :green_heart: |  javac  |   1m 21s |  the patch passed  |
| :green_heart: |  checkstyle  |   0m 34s |  The patch passed checkstyle in hbase-client  |
| :green_heart: |  checkstyle  |   1m 24s |  hbase-server: The patch generated 0 new + 6 unchanged - 2 fixed = 6 total (was 8)  |
| :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| :green_heart: |  shadedjars  |   5m  2s |  patch has no errors when building our shaded downstream artifacts.  |
| :green_heart: |  hadoopcheck  |  17m 18s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2.  |
| :green_heart: |  javadoc  |   0m 56s |  the patch passed  |
| :green_heart: |  findbugs  |   5m 52s |  the patch passed  |
||| _ Other Tests _ |
| :green_heart: |  unit  |   1m 49s |  hbase-client in the patch passed.  |
| :broken_heart: |  unit  | 261m  5s |  hbase-server in the patch failed.  |
| :green_heart: |  asflicense  |   1m  9s |  The patch does not generate ASF License warnings.  |
|  |   | 331m 55s |   |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.client.TestSnapshotTemporaryDirectoryWithRegionReplicas |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.4 Server=19.03.4 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/5/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/732 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 4f896a87f6cf 4.15.0-66-generic #75-Ubuntu SMP Tue Oct 1 05:24:09 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-732/out/precommit/personality/provided.sh |
| git revision | master / a881678ef4 |
| Default Java | 1.8.0_181 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/5/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/5/testReport/ |
| Max. process+thread count | 4933 (vs. ulimit of 10000) |
| modules | C: hbase-client hbase-server U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-732/5/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.11.0 https://yetus.apache.org |


This message was automatically generated.

**Pull request merged**